metrics/utils/video_align_method.py

```diff
@@ -41,3 +41,21 @@ def frame_align(self, src_video_info : VideoInfo, dst_video_info : VideoInfo):
             fp_new_video = self.change_video_fps_by_ffmepg(src_video_info, new_fps)
 
         return fp_new_video
+
+
+class VideoAlignMethodOcr(VideoAlignMethod):
+    def __init__(self):
+        super(VideoAlignMethodOcr, self).__init__()
+        self.align_method_name = "ocr"
+        self.file_dir = os.path.dirname(os.path.abspath(__file__))
+        self.file_path = os.path.join(self.file_dir, "ocr_frame_align.sh")
+
+    def frame_align(self, src_video_info : VideoInfo, dst_video_info : VideoInfo):
+
+        fp_new_src_video = NamedTemporaryFile('w+t', suffix=".%s" % (src_video_info.format_abbreviation))
```

Comment:
`fp` or `fd`?

Reply:
It should be a file-like object. So I will change all to fo_* instead of fp_* or fd_* in video evaluation. And I will create another PR to refactor others in network and audio evaluation.
- https://docs.python.org/3/library/tempfile.html?highlight=namedtemporaryfile#tempfile.TemporaryFile